configure.ac

```diff
@@ -902,6 +902,7 @@ for option in $enable_fast ; do
     esac
 done
 IFS="$save_IFS"
+export enable_fast
```

Comment:
Why export?

Reply:
In my tests, this wasn't visible in the subconfigure.m4 without export. I can try again without and verify.

Reply:
Looks like this does work with the export. I'll remove it.